pkg/util/test/util.go

```diff
@@ -277,32 +280,118 @@ func ValidateStorageCluster(
 		return err
 	}
 
+	k8sVersion, err := coreops.Instance().GetVersion()
+	if err != nil {
+		return err
+	}
+
+	if isPVCControllerEnabled(cluster) {
+		if err = validatePodsByName(cluster.Namespace, "portworx-pvc-controller", timeout, interval); err != nil {
+			return err
+		}
+
+		if err = validateImageTag(k8sVersion.String(), cluster.Namespace, map[string]string{"name": "portworx-pvc-controller"}); err != nil {
```

Comment:
We should ideally validate the complete image name. If people are using custom image registry, we should validate that we well with the tag.
Let's do that as part of a new change. For now this is fine.